packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -230,6 +230,7 @@ class WebView extends StatefulWidget {
     this.userAgent,
     this.initialMediaPlaybackPolicy =
         AutoMediaPlaybackPolicy.require_user_action_for_all_media_types,
+    this.transparent = false,
```

Comment:
Sorry to throw a wrench into the design aspect at this point, but I don't think we should add this; it will interact badly with implementing https://github.com/flutter/flutter/issues/37697 unless we make a breaking change, which is painful for everyone.

Is there a reason this can't just be a `backgroundColor` paramater that includes an alpha channel in the color specification?